ppm_telegram_bot/routers/telegram.py

```diff
@@ -1,41 +1,65 @@
+"""
+Endpoint for telegram webhook.
+
+One way to receive events from telegram is webhook: for each event (e.g. somebody writes to the bot or tap the button)
+telegram sends to us data about events. We should return "200 OK" and then do processing of new event.
+
+More info about webhook mechanism: https://core.telegram.org/bots/webhooks
+"""
 from secrets import compare_digest
+from typing import (
+    Dict,
+    Any,
+)
 
 from aiogram import Dispatcher
 from aiogram.types import Update
 from fastapi import (
     APIRouter,
     Depends,
+    Body,
 )
 from starlette.exceptions import HTTPException
-from starlette.requests import Request
 from starlette.responses import Response
 from starlette.status import HTTP_200_OK
-from ppm_telegram_bot.telegram import (
-    bot,
-    dispatcher,
+from ppm_telegram_bot.dependencies import (
+    bot_dispatcher,
+    telegram_bot,
 )
-from ppm_telegram_bot.dependencies import bot_dispatcher
 from ppm_telegram_bot.settings import settings
 
 router = APIRouter()
 
 
-@router.post("/telegram/webhook/{secret}")
+@router.post("/webhook/{secret}")
 async def telegram_webhook(
-    secret: str, request: Request, bot_dispatcher: Dispatcher = Depends(bot_dispatcher)
+    secret: str,
```

Comment:
[SecretStr](https://pydantic-docs.helpmanual.io/usage/types/#secret-types)?

Reply:
Why? 🤔 

SecretStr just has hidden when you try to print it or I missed something?

Reply:
Also, I moved this functionality to external library as `Path` object: 
https://github.com/b0g3r/fastapi-security-telegram-webhook/blob/21cc221d7aaa7180eb05876a7ab0680708413b3d/fastapi_security_telegram_webhook/security.py#L21-L25

Reply:
That's true, it can leak only if you print it or something. But it's still a secret. And you never know when and with what data logging can get through code review and then leak some sensitive data. Shit happens, and if you can cover a potential hole prematurely, it wouldn't hurt, I reckon.

Reply:
It's harder than you think: it isn't str or secrestr, it's Path object

Reply:
Didn't have time to look at the `fastapi-security-telegram-webhook` code, so, not sure what it does :thinking:

Reply:
It's just FastAPI path object

Reply:
We secure webhook via the token in the URL path, so we need to explicitly tell FastAPI that secret isn't from query or params